crates/zksync/core/src/vm/tracer.rs

```diff
@@ -139,9 +142,19 @@ impl CheatcodeTracer {
     ) -> Self {
         CheatcodeTracer { mocked_calls, expected_calls, call_context, result, ..Default::default() }
     }
+
+    /// Returns true if the given `target`'s code storage should result empty
+    fn is_code_expected_empty(&self, target: Address) -> bool {
+        target == foundry_common::HARDHAT_CONSOLE_ADDRESS || target == self.call_context.tx_caller
+    }
+
+    /// Load the specified storage slot
+    pub fn sload<S: ReadStorage>(ptr: StoragePtr<S>, key: StorageKey) -> StorageValue {
+        ptr.borrow_mut().read_value(&key)
+    }
```

Comment:
Not entirely convinced on this function. It replaces a one liner for another, plus forced EVM terminology onto zksync's `StorageView`, which isn't exactly necessary in my opinion.